CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 All notable changes to this project will be documented in this file.
 
+## [2.0.1](https://github.com/voxpupuli/modulesync/tree/2.0.1) (2020-10-06)
+
+[Full Changelog](https://github.com/voxpupuli/modulesync/compare/2.0.0...2.0.1)
+
+**Fixed bugs:**
+
+- Use remote\_branch for PRs when specified [\#194](https://github.com/voxpupuli/modulesync/pull/194) ([raphink](https://github.com/raphink))
+
+**Closed issues:**
+
+- Please adjust the dependency to puppet-blacksmith's current version [\#196](https://github.com/voxpupuli/modulesync/issues/196)
```

Comment:
I normally skip these if a PR was merged to address them (as it is now)

Reply:
jep, updated it